src/main/java/com/passbook/sparkeighteen/service/UserService.java

```diff
@@ -37,4 +39,24 @@ public SignUpResponse signUp(@NonNull final SignUpRequest signUpRequest) throws
                 .message("Sign-up is successful.")
                 .build();
     }
+
+    public LoginResponse login (final LoginRequest loginRequest) throws Exception {
+        final Optional<UserEntity> byEmail =
```

Comment:
Don't use variable names autogenerated by IDE, they mostly don't make sense

Reply:
Okay sir, Next time will give our own name to variable :)

Reply:
Please correct these in next commit